docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,118 @@
+accessing the snapshot repository.
+
+With {search-snaps} you may be able to halve your cluster size without
+increasing the risk of data loss or reducing the amount of data exposed to
+searches. Put differently, {search-snaps} may allow you to expose twice as much
+data to searches for a given cluster size.
+
+=== Using {search-snaps}
+
+A {search-snap} can be searched just like any other index. Since there is a
+copy of the {search-snap} on the nodes in the cluster its searches will perform
+similarly to searches on a regular index.
+
+The shards of {search-snaps} are allocated just like shards of any other index.
+You can, for instance, use <<shard-allocation-filtering>> to restrict these
+shards to a subset of your nodes.
+
+Normally you will manage {search-snaps} via the <<ilm-searchable-snapshot,
+searchable snapshots ILM action>> which automatically and transparently
+converts your index into a {search-snap} when it reaches the `cold` ILM phase.
+If you already have some snapshots that you want to search, you can also use
+the <<searchable-snapshots-api-mount-snapshot, mount snapshot API>> to manually
+mount them as {search-snaps}.
+
+You must not delete a snapshot while any of its indices are mounted as a
+{search-snap}. However, most snapshots are taken as a backup and contain a
+large number of indices, most of which will not be mounted as searchable
+snapshots. If you mount just one of these indices then you may not delete that
+snapshot. Therefore we recommend that you use the <<clone-snapshot-api, clone
+snapshot API>> to cheaply create a clone of a snapshot where the clone contains
+just the index you want to mount. This will mean you can delete the
+multiple-index snapshot, reducing the size of your snapshot repository, without
+losing access to any mounted indices.
+
+We recommend that you <<indices-forcemerge, force-merge>> indices to a single
+segment per shard before mounting them as {search-snaps}. Each read from a
+snapshot repository takes time and costs money, and the fewer segments there
+are the fewer reads are needed to restore the snapshot.
+
+By default a {search-snap} index has `number_of_replicas` set to `0`. You can
+increase the number of replicas if desired, for instance if you want to perform
+more concurrent searches of these shards.
+
+If a node fails while holding some shards of a zero-replica {search-snap} then
+there will be a brief window of time before {es} allocates these shards
+elsewhere.  During this window of time the cluster health will not be `green`
+and searches that hit these shards will fail or return partial results.
+
+[TIP]
+====
+{search-snaps-cap} are ideal for managing a large archive of historical data.
+Historical information is typically searched less frequently than recent data
+and therefore may not need replicas for their performance benefits.
+
+You can use <<async-search>> with {search-snaps}, which is especially useful
+for more complex or time-consuming searches.
```

Comment:
```suggestion
{search-snaps-cap} are ideal for managing large archives of historical data.
Historical information is typically searched less frequently than recent data
and performance is less important.

For more complex or time-consuming searches, you can use <<async-search>> with {search-snaps}.
```

Reply:
I'd rather keep this wording as-is:

- "performance is less important" suggests to me that there's a general performance penalty for using searchable snapshots -- in fact the only drawback most of the time is the lack of replicas.

I'll apply the change to the wording re. async searches separately.